ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -66,6 +66,7 @@ function build_push_operator_images {
 
   # Build and push operator image
   oc new-build --binary --strategy=docker --name ${OPERATOR} --to=${IMAGE_TAG_BASE}:${IMAGE_TAG} --push-secret=${REGISTRY_SECRET} --to-docker=true
+  oc set build-secret --pull bc/${OPERATOR} ${DOCKER_REGISTRY_SECRET}
```

Comment:
we probably don't need in the next 2 builds, but shouldn't we add there too?

Reply:
Hey Douglas,

If we look at logs[1] we only pull golang from docker.io, bundle and index don't pull anything from docker.io. So, its not really needed in the bundle/index build.


[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37285/rehearse-37285-pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy-kuttl/1635685969749348352/artifacts/keystone-operator-build-deploy-kuttl/openstack-k8s-operators-build/build-log.txt